src/test/java/apiStepsDefs/ApiStepsDefs.java

```diff
@@ -0,0 +1,55 @@
+package apiStepsDefs;
+
+import io.cucumber.java.en.*;
+
+import org.junit.Assert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Cookie;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import utils.ApiHelper;
+
+import java.util.concurrent.TimeUnit;
+
+import static constants.Constants.IMPLICITLY_WAIT_TIMEOUT;
+import static driver.CapabilitiesHelper.getChromeOptions;
+
+public class ApiStepsDefs {
+
+    ApiHelper apiHelper = new ApiHelper();
```

Comment:
In real life there will be each object responsible for work with each resource.
E.g. Cart, User, Wishlist etc. It will not be just helper one for all